public/scripts/worktrollyUI.js

```diff
@@ -2,20 +2,35 @@ $("#createNewTaskButton").click(function() {
     newPage = "createNewTask";
     uiLoader();
     var date = new Date();
-    $("#creationDateCreateNewTask").html(date.getDate() + "/" + (date.getMonth() + 1) + "/" + date.getFullYear());
-
+    $("#creationDateCreateNewTask").html(
+        date.getDate() + "/" + (date.getMonth() + 1) + "/" + date.getFullYear()
+    );
```

Comment:
Indentation issue please talk to shivam for js formatter extension, and fix them

Reply:
ok